src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/InterfaceDispatchMapIndirectionNode.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    class InterfaceDispatchMapIndirectionNode : EmbeddedObjectNode, ISymbolNode
```

Comment:
Yes that looks very do-able. I'll save this for a follow up check-in since this is just a check point in a batch of work I'm doing.